src/Sylius/Component/Report/Model/Report.php

```diff
@@ -88,11 +81,9 @@ public function getName()
     }
 
     /**
-     * Sets the value of name.
-     *
-     * @param string $name the name
+     * @param string $name
```

Comment:
We keep removing fluent interfaces and redundant comments in modified parts of code now, as fluency should be applied only where really needed and doing it before releasing stable version is the only choice :)